integration/grpc_test.go

```diff
@@ -0,0 +1,134 @@
+package integration
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"os"
+	"time"
+
+	"github.com/containous/traefik/integration/helloworld"
+	"github.com/containous/traefik/integration/try"
+	"github.com/go-check/check"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var LocalhostCert = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
Can we not re-use any of the other integration test certificates, instead of hard coding another here?